app/[lang]/(board)/board/[slug]/layout.tsx

```diff
@@ -0,0 +1,125 @@
+"use client"
+import { useEffect, useState } from "react"
+import { useRouter, useSearchParams } from "next/navigation"
+import { User, getCurrentUser } from "@/lib/api/session"
+import Loading from "../../../loading"
+import { getDictionary } from "../../../dictionaries"
+import Link from "next/link"
+import { ManualProxyForm } from "@/components/manual-proxy-form"
+import { MainNav } from "@/components/main-nav"
+import { boardConfig } from "@/config/board"
+import { Locale } from "@/i18n-config"
+import { LanguageToggle } from "@/components/language-toggle"
+import { Separator } from "@radix-ui/react-select"
+import { ModeToggle } from "@/components/mode-toggle"
+import { cn } from "@/lib/utils"
+import { buttonVariants } from "@/components/ui/button"
+import { SiteFooter } from "@/components/site-footer"
+import React from "react"
+
+interface boardLayoutProps {
+    children: React.ReactNode
+    params: {
+      lang: Locale
+    }
+  }
+  
+export interface BoardChildComponentProps {
+  excludeHeader?: boolean
+  excludeFooter?: boolean
+}
+
+export default function BoardLayout({ children, params: { lang } }: boardLayoutProps) {
+    const [user, setUser] = useState<User | null>(null)
+    const [isLoading, setIsLoading] = useState(true)
+    const [dict, setDict] = useState<Record<string, any> | null>(null)
+    const router = useRouter()
+    const searchParams = useSearchParams()
+    const refreshToken = searchParams.get("refresh_token")
```

Comment:
We don't need refresh_token, noredirect, mode, eType, etc.
Those are query params used for managing redirects when applying with apply.embloy.com, but not needed for the job board